### ISSUE TITLE
Navigation Portlet Style

### ISSUE THREAD

**Issue by skleinfeldt** [PSUEducationalEquity/plonetheme.eeq#18]
As a user I can see navigation (where needed) in the top right sidebar portlet content area themed appropriately throughout the site. (Does not include homepage or office landing pages.)

**Comment by witekdev** [comment]
@MBrahosky @paulrentschler 
Implemented. My local site looks like this:

![Screenshot from 2021-06-07 10-10-52](https://user-images.githubusercontent.com/883619/121055374-7b851e00-c7bd-11eb-8f71-0a0f4c19e906.png)

![Screenshot from 2021-06-07 10-10-40](https://user-images.githubusercontent.com/883619/121055396-7fb13b80-c7bd-11eb-9a49-8f7d1bdc0e9c.png)

**Comment by paulrentschler** [comment]
Is it possible with Diazo rules to easily switch the order of parent and local portlets on the right side so that the "All In" portlet appears below the navigation portlet. I know we can block parent portlets and recreate the navigation portlet to achieve this, just wondering if the switch is easily done everywhere at once with Diazo. If it is not, then we will use the block parents/recreate method.

**Comment by skleinfeldt** [comment]
Good question to ask in meeting tomorrow @paulrentschler if you haven't heard before then.

**Comment by alecpm** [comment]
I don't believe there are any good CSS hooks for detecting local vs acquired portlets, however it should be possible to move the navigation portlet to the bottom of the stack whenever it's at the top. Not sure whether that solves the problem though.

**Comment by paulrentschler** [comment]
We are going to leave this as is and use Plone to block parent portlets and recreate as necessary.
